tests/panels/test_sql.py

```diff
@@ -225,12 +225,16 @@ def test_json_param_conversion(self):
         connection.vendor == "postgresql", "Test valid only on PostgreSQL"
     )
     def test_tuple_param_conversion(self):
+        """
+        For psycopg3 You cannot use IN %s with a tuple
+        https://www.psycopg.org/psycopg3/docs/basic/from_pg2.html#you-cannot-use-in-s-with-a-tuple
+        """
         self.assertEqual(len(self.panel._queries), 0)
 
         list(
             PostgresJSON.objects.raw(
-                "SELECT * FROM tests_postgresjson WHERE field ->> 'key' IN %s",
-                [("a", "b'")],
+                "SELECT * FROM tests_postgresjson WHERE field ->> 'key' = ANY(%s)",
+                [["a", "b'"]],
```

Comment:
I'm not sure this test should change. It should be updated to expectedly fail with psycopg3.

Reply:
Ah sorry, I suggested only keeping the new version since that one would work both with psycopg2 and psycopg3. If you want to keep the old version around as well and just run one of them depending on the installed psycopg version that's fine too.

Reply:
I think I'm more comfortable with that since this is a regression test (https://github.com/jazzband/django-debug-toolbar/pull/1722) and the updated version avoids that test case altogether

Reply:
I wasn't sufficiently aware of that. I agree then. Sorry @nofalx